packages/example-app-base/src/components/App/App.tsx

```diff
@@ -20,13 +21,23 @@ const getClassNames = classNamesFunction<IAppStyleProps, IAppStyles>();
 export class AppBase extends React.Component<IAppProps, IAppState> {
   public state: IAppState = { isMenuVisible: false };
   private _classNames: IProcessedStyleSet<IAppStyles>;
+  private _baseUrl: string;
+
+  constructor(props: IAppProps) {
+    super(props);
+
+    const doc = getDocument();
+    this._baseUrl = doc ? document.location.href : '';
+  }
 
   public render(): JSX.Element {
     const { appDefinition, styles, responsiveMode = ResponsiveMode.xLarge, theme } = this.props;
     const { customizations } = appDefinition;
     const { isMenuVisible } = this.state;
 
-    const classNames = (this._classNames = getClassNames(styles, { responsiveMode, theme }));
+    const onlyExamples = showOnlyExamples(this._baseUrl);
```

Comment:
Sorry to keep picking on this, but I think the part where you get `document.href` should also be in the helper. It would probably be safe to do the check in the constructor and store the result in a member, since this query param should be set on page load and never updated.

Reply:
No need to apologize. Thanks for the feedback. Done!